src/tutorial/counter-app/tui.md

```diff
@@ -28,13 +28,18 @@ example, here's the `init` method:
 
 ```rust
 {{#include ./ratatui-counter-app/src/tui.rs:tui_init}}
+}
 ```
 
 This is essentially the same as the `startup` function from before. One important thing to note that
 this function can be used to set a panic hook that calls the `reset()` method.
 
 ```rust
+impl tui {
+    // --snip--
+
 {{#include ./ratatui-counter-app/src/tui.rs:tui_exit}}
+}
```

Comment:
I think we want a `// --snip--` before and after the code block. Also, I think the `// --snip--` should be 2 spaces to align with the code included. And this should be `impl Tui`.

```suggestion
impl Tui {
  // --snip--
{{#include ./ratatui-counter-app/src/tui.rs:tui_exit}}
  // --snip--
}
```